antiplag/antiplag/views.py

```diff
@@ -28,22 +30,45 @@ class FileDetail(APIView):
     parser_classes = (MultiPartParser, FormParser)
 
     def post(self, request, *args, **kwargs):
-
-        submission_serializer = SubmissionSerializer(data=request.data)
+        submission_serializer = SubmissionSerializer(data={"status": Submission.SubmissionStatus.PENDING})
         if submission_serializer.is_valid():
             submission = submission_serializer.save()
-            for file in request.FILES.getlist("files"):
-                doc_serializer = DocumentSerializer(data={"file": file})
+
+            if request.META['HTTP_TYPE'] == Document.DocumentType.FILE:
```

Comment:
Use https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type, not our interal ENUM